### PR TITLE
Add swiper support, fix typos and drop dependencies

### PR DESCRIPTION
**Pull request by drym3r**
As the title says. More info on the commits. If you don't wan't to drop the dependencies, you can just drop the last commit, swipper support is added as dependency like the other three.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,10 +18,10 @@ be invoked. Of course you can customize the above behaviour.
...
 
 * Emacs 24 or higher
-* [ace-jump-mode](https://github.com/winterTTr/ace-jump-mode)
-* [avy](https://github.com/abo-abo/avy)
-* [helm-swoop](https://github.com/ShingoFukuyama/helm-swoop)
-* [swiper](https://github.com/abo-abo/swiper/)
+* [ace-jump-mode](https://github.com/winterTTr/ace-jump-mode) or [avy](https://github.com/abo-abo/avy)
+* [helm-swoop](https://github.com/ShingoFukuyama/helm-swoop) or [swiper](https://github.com/abo-abo/swiper/)
+
+You must install this packages manually.
```
Comment (mdorman) [inline README.md]: Don't you mean "these packages"?

**Comment by mdorman** [comment]
I like this idea; I use `swiper` rather than `helm-swoop` myself, so not having that dependency would be quite nice.

File: README.md (hunk 1 above)
Comment (drym3r) [inline README.md]: Oh, sure. I just changed.

**Comment by drym3r** [comment]
Ping.

**Comment by tam17aki** [comment]
Hi,
Thank you for using the ace-isearch and sorry for late reply.
I'll check your request in this week.

Best regards,
Akira

**Comment by drym3r** [comment]
Great, just wanted to see if you were interested, no hurries

**Comment by tam17aki** [comment]
It seems good and I will try to test it with your request reflected.
Please wait until finished.

Best regards,
Akira

**Pull request merged**